root/etc/e-smith/templates/etc/httpd/conf.d/nethserver.conf/10ssl

```diff
@@ -3,7 +3,57 @@
 #
 # Additional directives for SSL. See also upstream ssl.conf.
 #
-{ ($pki{ChainFile} && -e $pki{ChainFile})? '' : '# '}SSLCertificateChainFile { $pki{ChainFile} }
-{ $httpd{SSLCipherSuite} ? '' : '# '}SSLCipherSuite {$httpd{SSLCipherSuite}}
+{
+ my $tlsPolicy = $tls{'policy'} || 'legacy';
+ my $chainfile = ($pki{ChainFile} && -e $pki{ChainFile})? '': '#';
+ my $ciphersuite = ($httpd{SSLCipherSuite})? '' : '#';
+
+ if ($tlsPolicy eq 'legacy') {
+    $OUT = $chainfile . "SSLCertificateChainFile $pki{ChainFile}\n";
+    $OUT .= $ciphersuite . "SSLCipherSuite $httpd{SSLCipherSuite}\n";
+ }
+ elsif ($tlsPolicy eq '20180330') {
+    $OUT .= << 'EOF';
+<VirtualHost _default_:443>
```

Comment:
I'd like (if possible) to split the `<VirtualHost>` section in a separate fragment. Also I'd like to split each policy implementation in a separate fragment (also for Dovecot and other packages).

- 10ssl: define `SSLCertificate*` parameters in the global scope
- 10tls_policy_legacy: IF policy = legacy then apply old policy
- 10tls_policy_20180330: IF policy = 20180330 then apply strong ciphers and protocols definition
- 50ssl_default_virtualhost: `<VirtualHost *:443>` (* is alias for _default_) with Log files, and anything else you can find with `grep -v ^# /etc/httpd/conf.d/ssl.conf` excluding TLS related settings, already set in the global scope